src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/client/payhub/PayHubClientErrorDecoder.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.client.payhub;
+
+import feign.Response;
+import feign.codec.ErrorDecoder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.HttpServerErrorException;
+
+import java.util.Optional;
+
+public class PayHubClientErrorDecoder implements ErrorDecoder {
+
+    private static final Logger log = LoggerFactory.getLogger(PayHubClientErrorDecoder.class);
```

Comment:
not that but let's keep it consistent upper case for static final variables

Reply:
removed